src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1114,7 +1114,7 @@ function GlobeControls(view, target, radius, options = {}) {
 
             if (point) {
                 animatedScale = 0.6;
-                this.setCameraTargetPosition(point, this.isAnimationEnabled());
+                this.setCameraTargetPosition(point, true);
```

Comment:
This was for testing right? If so should be remoed

Reply:
I added comment, there's conflict simple/double click without animation. I'll fix it in the refacto.